R/slice.R

```diff
@@ -395,7 +397,7 @@ check_slice_dots <- function(..., n, prop, error_call = caller_env()) {
     dots <- enquos(...)
 
     if (length(dots) == 1L && names2(dots)[1] == "") {
-      slice_call <- error_call$.Generic
+      slice_call <- caller_call()[[1]]
```

Comment:
```suggestion
      slice_call <- frame_call(frame = error_call)[[1]]
```

Is this _slightly_ more correct? Probably doesn't matter much, but we do have `error_call` as an argument here so in theory the direct caller env isn't always the right one to get the `slice_call` from

Reply:
Added by hand, thanks!